model/utils.go

```diff
@@ -334,7 +335,10 @@ func NewRandomTeamName() string {
 // The resulting entropy will be (5 * length) bits.
 func NewRandomString(length int) string {
 	data := make([]byte, 1+(length*5/8))
-	rand.Read(data)
+	_, err := rand.Read(data)
+	if err != nil {
+		mlog.Error("Reading random bytes failed", mlog.Err(err))
+	}
```

Comment:
Let's leave it for now. I'd really want to avoid logging in the `model` package. One thing I noticed is that this function is mainly used in tests, and the places it's used in non-test code are handful. We can make an internal helper which returns an error and avoid calling this, and let the tests call this. 0/5, up to you.

Reply:
If it's just for tests, let's drop the error.

Reply:
Go for it :+1: